go/vt/sqlparser/ast.go

```diff
@@ -445,6 +445,7 @@ type Select struct {
 	OrderBy       OrderBy
 	Limit         *Limit
 	Lock          string
+	Into          Columns
```

Comment:
These are not technically `Columns`, right? Check out the other comment below about considering a new rule for `variable_list` instead of reusing `column_list` and you might be able to clean this up, too. For example, this might could just be a simple `[]string` and that would also simplify the logic in the `Format` method to save you a line or two.

Reply:
Right! I think adding variable_list with `Variables` struct is good idea as you suggested in the other comment, and Into variables also are case insensitive so ColIter works. It's like partitions struct, an alias of columns, so its Format can be used directly in Select Format method.